libraries/AP_Scripting/applets/ExpressLRS.md

```diff
@@ -0,0 +1,87 @@
+# Bidirectional ExpressLRS Telemetry Demo
+
+Tested with: Arducopter 4.4.0-DEV + ExpressLRS 3.1.2 100Hz Full 1:2 + EdgeTX 2.8.0
+
+NOTE: the first data byte _NEEDS_ to be 0x00 or 0xc8 in EdgeTX `crossfireTelemetryPush(command, data)`, otherwise no data is received. (This appears to be a bug/feature in ExpressLRS 3.1.2 and/or EdgeTX 2.8.0)
```

Comment:
this is not a bug, it's by design, 0x00 is a broadcast address and 0xC8 is flightcontroller device address, so in your testing you should target the flight controller, 0x00 is probably too broad